src/control/server/ctl_ranks_rpc_test.go

```diff
@@ -774,14 +786,16 @@ func TestServer_CtlSvc_StartRanks(t *testing.T) {
 			expResults: []*sharedpb.RankResult{},
 		},
 		"instances already started": {
-			req: &ctlpb.RanksReq{Ranks: "0-3"},
+			req:     &ctlpb.RanksReq{Ranks: "0-3"},
+			timeout: time.Second,
 			expResults: []*sharedpb.RankResult{
 				{Rank: 1, State: msReady},
 				{Rank: 2, State: msReady},
 			},
 		},
 		"instances get started": {
 			req:              &ctlpb.RanksReq{Ranks: "0-3"},
+			timeout:          5 * time.Second,
```

Comment:
Guess I'm still a little confused. You mentioned they would need some amount of time, but if we expect the test to be deterministic, i.e. it gets there eventually, why pass a timeout at all?

Reply:
good point, fixed